controllers/barbican_controller.go

```diff
@@ -89,6 +89,8 @@ func (r *BarbicanReconciler) GetLogger(ctx context.Context) logr.Logger {
 //+kubebuilder:rbac:groups=rabbitmq.openstack.org,resources=transporturls,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=batch,resources=jobs,verbs=get;list;watch;create;update;patch;delete;
 //+kubebuilder:rbac:groups=mariadb.openstack.org,resources=mariadbdatabases,verbs=get;list;watch;create;update;patch;delete;
+//+kubebuilder:rbac:groups=k8s.cni.cncf.io,resources=network-attachment-definitions,verbs=get;list;watch
```

Comment:
Don't we need the "delete" or "update" verbs as well?

Reply:
No, because we're not actually creating/updating/deleting the NAD resources.  We're only checking that they exist when someone specifies an attachment in the CR specs, i.e.

```
    networkAttachments:
      - storage
```

The NADs themselves are created manually by the user beforehand i.e. https://github.com/openstack-k8s-operators/architecture/blob/main/va/hci/stage3/netattach_storage.yaml